instrumentation/couchbase/couchbase-3.1.6/metadata.yaml

```diff
@@ -0,0 +1,4 @@
+description: >
+  Couchbase instrumentation is owned upstream. This Couchbase instrumentation manipulates the
```

Comment:
I think we don't need to provide the details on how exactly the instrumentation works as it requires the reader to know how tracing works in couchbase to make sense of it. Maybe we could be a bit more vague and just say we automatically enable the instrumentation that is provided by the couchbase.

> Couchbase instrumentation is owned by the Couchbase project. This instrumentation automatically configures the instrumentation provided by the Couchbase.